extensions/cornerstone/src/initWADOImageLoader.js

```diff
@@ -4,15 +4,41 @@ import dicomParser from 'dicom-parser';
 
 //import { initWebWorkers } from './utils/index.js';
 
-cornerstoneWADOImageLoader.external.cornerstone = cornerstone;
-cornerstoneWADOImageLoader.external.dicomParser = dicomParser;
-
-cornerstoneWADOImageLoader.configure({
-  beforeSend: function(xhr) {
-    /*const headers = OHIF.DICOMWeb.getAuthorizationHeader();
-
-    if (headers.Authorization) {
-      xhr.setRequestHeader('Authorization', headers.Authorization);
-    }*/
-  },
-});
+let initialized = false;
+
+function initWebWorkers() {
+  const config = {
+    maxWebWorkers: Math.max(navigator.hardwareConcurrency - 1, 1),
+    startWebWorkersOnDemand: true,
+    taskConfiguration: {
+      decodeTask: {
+        initializeCodecsOnStartup: false,
+        usePDFJS: false,
+        strict: false,
+      },
+    },
+  };
+
+  if (!initialized) {
+    cornerstoneWADOImageLoader.webWorkerManager.initialize(config);
+    initialized = true;
+  }
+}
+
+export default function initWADOImageLoader(UserAuthenticationService) {
```

Comment:
Maybe we expose webworker configs here as well? We discussed `convertFloat32ToInt`

Reply:
I'll do this in another PR